geiger/src/graph/Cell.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @typedef {import('../graph/Graph').default} Graph */
+
+/**
+ * A Cell could be Node or Edge. They all have property `graph`.
+ * @class Cell
+ */
+export default class Cell {
+  constructor() {
+    this.graph = null;
+  }
+
+  /**
+   * A shortcut to add cell to graph.
+   * @param {!Graph} graph
+   * @return {!Cell}
+   */
+  addTo(graph) {
```

Comment:
...then we don't need this method at all.

Reply:
Removed.